chart/skywalking/templates/es-init.job.yaml

```diff
@@ -91,18 +91,39 @@ spec:
         - name: {{ $key }}
           value: {{ $value | quote }}
         {{- end }}
+        {{- range .Values.oap.secretKeyRefs }}
+        - name: {{ .name }}
+          valueFrom:
+            secretKeyRef:
+              name: {{ .secretName }}
+              key: {{ .key }}
+        {{- end }}
         volumeMounts:
           {{- if (.Files.Glob "files/conf.d/oap/**") }}
-            {{ range $path, $bytes := .Files.Glob "files/conf.d/oap/**" }}
-            - name: skywalking-oap-override
-              mountPath: {{ print "/skywalking/config/" ($path | replace "files/conf.d/oap/" "") }}
-              subPath: {{ $path | replace "files/conf.d/oap/" "" | b64enc | replace "=" "-" }}
-            {{- end }}
-            {{- end }}
+          {{ range $path, $bytes := .Files.Glob "files/conf.d/oap/**" }}
+          - name: skywalking-oap-override
+            mountPath: {{ print "/skywalking/config/" ($path | replace "files/conf.d/oap/" "") }}
+            subPath: {{ $path | replace "files/conf.d/oap/" "" | b64enc | replace "=" "-" }}
+          {{- end }}
+          {{- end }}
+          {{- range .Values.oap.secretVolumes }}
+          - name: {{ .name }}
+            readOnly: true
+            mountPath: {{ .mountPath | default "/skywalking/certs" }}
```

Comment:
defaulting to "/skywalking/certs" hints that the secret is limited to store certs and keys, how about leaving it alone?